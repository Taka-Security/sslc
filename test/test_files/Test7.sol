contract A {
  struct StructA {
    bool var1;
    uint16 var2;
    bytes32 var3;
    bytes32 var4;
    string var5;
    uint8 var6;
  }
  struct StructB {
    uint88 var1;
    StructA var2;
    uint88 var3;
  }
}