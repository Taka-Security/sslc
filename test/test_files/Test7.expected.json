[
    {
        "file": "Test7.sol",
        "contract": "A",
        "struct": "StructA",
        "slotsSaved": 1,
        "timedout": false
    },
    {
        "file": "Test7.sol",
        "contract": "A",
        "struct": "StructB",
        "slotsSaved": 1,
        "timedout": false
    }
]